static/style.css

```diff
@@ -322,3 +322,7 @@ svg text {
 .data-stack:hover .success-bar {
   fill: #31b0d5;
 }
+
+.leaderboard-overview {
+  min-width: 1250px;
```

Comment:
What is this arbitrary value of `1250px` suppose to represent? Rather than using min-width, leverage Bootstrap's column breakpoints.